### PR TITLE
Rdoc-1919 update backup retention faq

### PR DESCRIPTION
**Pull request by serezhub**

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+Although [External Replication](../../../../studio/database/tasks/ongoing-tasks/external-replication-task) 
+and [Backup](../../../../client-api/operations/maintenance/backup/backup) 
+are both ongoing-tasks, they have different aims and behavior.  
+See [Backup Task -vs- External Replication Task](../../../../studio/database/tasks/ongoing-tasks/backup-task#backup-task--vs--replication-task).  
+
+---
+
+###Can I simply copy the database folder contents whenever I need to create a backup?  
+
+Simply copying the database folder is **not** a good substitute for RavenDB's backup procedures.
```
Comment (haludi) [inline Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:56]: Simply copying the database folder with continuously writing will probably end with corrupted data in the backup

File: Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 ###Is there a one-time backup?
 
-No. Backup is an on-going task and is meant to back your data up continuously.  
+No. Backup is an ongoing task and is meant to back your data up continuously.
```
Comment (haludi) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:24]: https://github.com/ravendb/docs/pull/1463/files#r830974930

File: Documentation/5.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+{NOTE/}
+
+---
+
+{PANEL: FAQ}
+
+###Is there a one-time backup?
+
+No. Backup is an ongoing task and is meant to back your data up continuously.
```
Comment (haludi) [inline Documentation/5.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:24]: https://github.com/ravendb/docs/pull/1463/files#r830974930

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+Learn how to change the [Retention Policy via API](../../../../client-api/operations/maintenance/backup/backup#backup-retention-policy).  
+Learn how to change the [Retention Policy via RavenDB Studio](../../../../studio/database/tasks/ongoing-tasks/backup-task#backup-task---retention-policy).
+
+---
+
+###Are there any locations that backup files should NOT be stored at?  
+
+It is recommended **not** to store backups on the same drive as your database data files, since -  
+
+* Reading from and writing to the same drive can slow down other database operations.
```
Comment (haludi) [inline Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:80]: This is not relevant while we writing the backup to the local server first

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+{NOTE/}
+
+---
+
+{PANEL: FAQ}
+
+###Is there a one-time backup?
+
+No. Backup is an ongoing task and is meant to back your data up continuously.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:24]: true. fixed

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown (hunk 1 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:56]: Yeah.  That is a far better explanation

File: Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown (hunk 2 above)
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:24]: This links to the same PR

File: Documentation/5.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown (hunk 3 above)
Comment (serezhub) [reply Documentation/5.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:24]: ditto

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown (hunk 4 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown]: fixed.  It always writes first locally, right?  I removed this bullet point.

File: Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown (hunk 2 above)
Comment (haludi) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown]: https://github.com/ravendb/docs/pull/1463#discussion_r830974930

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown (hunk 4 above)
Comment (haludi) [reply Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown]: Yes, we always writes the backup/snapshot locally first

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown [hunk 6]
```diff
@@ -21,17 +21,18 @@
...
-* If you wish, you can trigger it for immediate execution and then disable or delete the task.  
 * You can also use [Smuggler](../../../../client-api/smuggler/what-is-smuggler#what-is-smuggler) as an equivalent of a full backup for a single [export](../../../../client-api/smuggler/what-is-smuggler#export) operation.  
 
 ---
 
 ###How do I create a backup of my cluster configuration?  
 
-Only database contents can be backed up. 
-Cluster configuration and nodes setup can be easily re-created, no special backup procedure is needed for it.  
+Only database contents and indexes can be backed up.
```
Comment (haludi) [inline Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:34]: If you specify indexes you should specify all the parts - identities, compare exchange, subscription definition, and all ongoing task definitions.
Not that the backup contains only the definition of the index while snapshot contains also the actual data

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown [hunk 7]
```diff
@@ -75,11 +77,11 @@ Learn how to change the [Retention Policy via RavenDB Studio](../../../../studio
 
 ###Are there any locations that backup files should NOT be stored at?  
 
-It is recommended **not** to store backups on the same drive as your database data files, since -  
+It is recommended **not to store backups on the same drive as your database** data files, 
+since both the database and the backups would be exposed to the same risks.  
 
-* Reading from and writing to the same drive can slow down other database operations.  
-* Disk space can run low as backups start piling up.
```
Comment (haludi) [inline Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown]: You can keep it. 
We do write the backup first to the local server but we deleting it after that.

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown (hunk 6 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:34]: Good. I didn't know that it also backs these up.  Thanks for showing me how to check.

File: Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown (hunk 7 above)
Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown]: I'll leave "Disk space can run low..." but add "unless you configure the retention policy (with a link)."

Comment (serezhub) [reply Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown]: I put it back and added "...piling up unless you set a retention policy(link)."

Comment (haludi) [reply Documentation/4.2/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown]: I don't think "unless you set a retention policy" is relevant here (but that is just my humble opinion)

File: Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown [hunk 8]
```diff
@@ -29,24 +29,29 @@ you can also use [one-time manual backups](../../../../studio/database/tasks/bac
...
 * Revisions
 * Conflict configurations
 * Identities
 * Compare-exchange items
-* Subscriptions.  
+* Subscriptions  
 
-Cluster configuration and nodes setup can be [re-created](../../../../start/getting-started#installation--setup) 
-and [restored from backup](../../../../studio/server/databases/create-new-database/from-backup), so no special backup procedure is needed for it.  
+Note that **backups don't save ETL and replication** tasks.
```
Comment (haludi) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:47]: Why do you say so?
When you store from a backup you have the ongoing tasks as well
Actually, when restoring you can explicitly choose "Disable ongoing tasks after restore"

File: Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown [hunk 9]
```diff
@@ -29,24 +29,29 @@ you can also use [one-time manual backups](../../../../studio/database/tasks/bac
...
 
-Both binary "Snapshot" and json "Backup" types of backup tasks save: 
+You can run a [server-wide ongoing backup](../../../../studio/server/server-wide-backup) 
+which backs up each of the databases in your cluster.  
+What does it back up? Both binary "Snapshot" and json "Backup" types of backup tasks save: 
 
 * Database contents
-* Document extensions
-* Indexes
+* Document extensions (attachments, counters, time-series)
```
Comment (haludi) [inline Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:39]: In my humble opinion, all those details are not relevant here.
Just a simple answer to the question is enough, no?

Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown:40]: The question wasn't clear.  I changed the question to let people know what is backed up and how to go about rebuilding a cluster via restore

File: Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown (hunk 8 above)
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/client-api/operations/maintenance/backup/faq.markdown]: True.  We saw that all ongoing tasks are backed up by default.  I changed that in the document.

**Pull request merged**